lib/site_template/_config.yml

```diff
@@ -29,6 +29,15 @@ markdown: kramdown
 theme: minima
 gems:
   - jekyll-feed
-exclude:
-  - Gemfile
-  - Gemfile.lock
+
+# Exclude from processing.
+# The following items will not be processed, by default. Create a custom list
+# to override the default setting.
+# exclude:
```

Comment:
But we would just want it to be comments rather than an actual list.

I just worry about adding files to the exclude list and forgetting to document the change here. We'll just have to be cautious, I guess